### PR TITLE
Add selector-max-type rule

### PR DESCRIPTION
**Pull request by 343dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,5 +19,6 @@ Obviously Stylelint should be installed too.
 - [declarations-before-nesting](./docs/rules/declarations-before-nesting.md) — forces block rules be placed above the nested blocks.
 - [no-transition-all](./docs/rules/no-transition-all.md) — prevents use `all` value for `transition` & `transition-property` props.
 - [rgba-hex-color-only](./docs/rules/rgba-hex-color-only.md) — prevents use `rgba` function with non-hex color as a first argument.
+- [selector-max-type](./docs/rules/selector-max-type.md) — limit the number of type selectors in a selector (compatible with [bem-react-helper](https://github.com/igoradamenko/bem-react-helper)).
```
Comment (igoradamenko) [inline README.md:22]: 🤔 

This rule is not related to bem-react-helper. The helper is `b()` func in React components. 

I think you wanted to note here our [scss-vars-loader](https://github.com/funbox/scss-vars-loader).

File: src/rules/selector-max-type/utils/isStandardSyntaxSelector.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Check whether a selector is standard
+ *
+ * @param {string} selector
+ * @returns {boolean}
+ */
+// eslint-disable-next-line func-names
+module.exports = function (selector) {
+  // SCSS interpolation (compatible with bem-react-helper)
```
Comment (igoradamenko) [inline src/rules/selector-max-type/utils/isStandardSyntaxSelector.js:9]: Same here.

**Review by igoradamenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (343dev) [reply README.md]: Oh yes, of course. Thanks.

**Review by igoradamenko — APPROVED** [review]
👍

**Comment by igoradamenko** [comment]
Published as @funboxteam/stylelint-rules@2.2.0.

**Comment by igoradamenko** [comment]
Thank you! 🎉
